pytorch_lightning/loggers/file_logger.py

```diff
@@ -0,0 +1,180 @@
+"""
```

Comment:
File Logger may be a bit misleading, since tensor board generates a file as well :) Maybe rename this file to `csv.py` and the Logger to `CSVLogger`?